### DISCUSSION TITLE
Breaking items

### DISCUSSION THREAD

**Discussion by SimpleFoxOfficial** [BEEmod/BEE2-items#4123] [Feature Suggestions]
Destroyable bricks wall will be like glass, but it's bricks and it can be destroyed by explosions/like monitors.
Floor, that was in chapter 1, where you find first portal gun. Breaks, if you stay on him

**Comment by slambo311** [comment]
Not gonna lie, this would be great. The floor is pretty much the same as when you use triggers mixed with 90° panels, but it does seem cool. "Start Enabled" could make them not breakable, and getting an input would make them break instantly. Actually, why not send an output when it breaks?

**Reply by SimpleFoxOfficial** [reply]
yep. And the floor with start reversed will be in other style, or like panels in any ' (90,45 etc) will change the style

**Reply by SimpleFoxOfficial** [reply]
the Bricks wall can be a skin for breakble glass.

**Comment by Coke-Unity** [comment]
I think it would make more sense to implement this as a "Breakable Wall" items that would have multiple skins depending on the cube type, or maybe make it a timer so there can be many skins? it should have a Glass skin, a Portalable Surface skin, a Non-Portalable Surface skin, maybe a wood skin and a brick skin? Grating skin? the possibility.

And yes, you can place portals on the Portalable Surface skin.

The things slambo311 are also good. inputs and outputs.
